tendrl/ceph_integration/manager/crud.py

```diff
@@ -119,3 +123,14 @@ def delete(self, object_type, object_id):
             )
         else:
             raise NotImplementedError(object_type)
+
+    def sync_request_status(self, request):
+        count = 0
+        while count < 30:
```

Comment:
we need locking for pool and other ceph objects because we are waiting on  them here